src/rgw/rgw_op.cc

```diff
@@ -6981,7 +6981,10 @@ void RGWDeleteMultiObj::handle_individual_object(const rgw_obj_key& o, optional_
   del_op->params.marker_version_id = version_id;
 
   op_ret = del_op->delete_obj(this, y);
-  if (op_ret == -ENOENT) {
+  if (op_ret >= 0) {
+    ldpp_dout(this, 1) << "DELETED: bucket=" << bucket->get_name() << ", obj=" << obj->get_oid() << dendl;
+  } else if (op_ret == -ENOENT) {
```

Comment:
This is going to miss any errors other than ENOENT.  I think you just need an "else" here, with an embedded "if (op_ret == -ENOENT) op_ret = 0;" inside it.

Reply:
Thank you for your comments. It's right, I need to modify it.